### PR TITLE
Dilpreet | Stage | Create database for the UK region

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/86cuw1c1z

### CODE REVIEW

File: apps/web-giddh/src/app/app.module.ts [hunk 1]
```diff
@@ -110,7 +110,9 @@ if (!environment.production) {
         WindowRef,
         {
             provide: ServiceConfig,
-            useValue: { apiUrl: Configuration.ApiUrl, appUrl: Configuration.AppUrl, _ }
+            useValue: localStorage.getItem('Country-Region') === 'GB'
+                ? { apiUrl: Configuration.DevApiUrl, appUrl: Configuration.DevApiUrl, _ }
+                : { apiUrl: Configuration.ApiUrl, appUrl: Configuration.ApiUrl, _ }
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/app.module.ts]:
```suggestion
                : { apiUrl: Configuration.ApiUrl, appUrl: Configuration.AppUrl, _ }
```

File: apps/web-giddh/src/app/app.module.ts [hunk 2]
```diff
@@ -110,7 +110,9 @@ if (!environment.production) {
         WindowRef,
         {
             provide: ServiceConfig,
-            useValue: { apiUrl: Configuration.ApiUrl, appUrl: Configuration.AppUrl, _ }
+            useValue: localStorage.getItem('Country-Region') === 'GB'
+                ? { apiUrl: Configuration.DevApiUrl, appUrl: Configuration.DevApiUrl, _ }
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/app.module.ts]:
```suggestion
                ? { apiUrl: Configuration.DevApiUrl, appUrl: Configuration.AppUrl, _ }
```

File: apps/web-giddh/src/environments/environment.local.electron.ts [hunk 3]
```diff
@@ -7,6 +7,7 @@ export const environment: Environment = {
     showDevModule: true,
     AppUrl: 'http://localhost:4200/',
     ApiUrl: 'https://api.giddh.com/',
+    DevApiUrl: 'http://giddh-api-prod-g.eu-west-2.elasticbeanstalk.com/',
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/environments/environment.local.electron.ts]: use https url

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/app.module.ts (hunk 1 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/app.module.ts]: Done

File: apps/web-giddh/src/app/app.module.ts (hunk 2 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/app.module.ts]: Done

File: apps/web-giddh/src/environments/environment.local.electron.ts (hunk 3 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/environments/environment.local.electron.ts]: As confirmed with Ashish and Kriti, we are using this URL. However, if it's for HTTPS, we don't have a certificate for it.

**Review by dvCodeWorld — DISMISSED** [review]

File: apps/web-giddh/src/environments/environment.local.electron.ts (hunk 3 above)
Comment (ravinderwalkover) [reply apps/web-giddh/src/environments/environment.local.electron.ts]: @dilpreetsinghofficial will this work on test?

**Review by mohitsharma93 — APPROVED** [review]

**Pull request merged**